components/dashboard/src/settings/Integrations.tsx

```diff
@@ -756,9 +759,23 @@ export function GitIntegrationModal(
                             >
                                 <option value="GitHub">GitHub</option>
                                 <option value="GitLab">GitLab</option>
+                                <option value="BitbucketServer">Bitbucket Server</option>
                             </select>
                         </div>
                     )}
+                    {mode === "new" && type === "BitbucketServer" && (
+                        <InfoBox className="my-4 mx-auto">
+                            Support for OAuth2 in Bitbucket Server was{" "}
+                            <a
+                                target="_blank"
+                                href="https://confluence.atlassian.com/bitbucketserver/bitbucket-data-center-and-server-7-20-release-notes-1101934428.html"
+                                rel="noopener noreferrer"
+                                className="gp-link"
+                            >
+                                added in version 7.20
+                            </a>
+                        </InfoBox>
```

Comment:
**praise**: Great usage of the alert component here, @AlexTugarev! 🔝

Reply:
**suggestion**: What do you think of structuring the alert message with the pattern we've used elsewhere in the product?

| BEFORE | AFTER |
|-|-|
| <img width="1440" alt="Screenshot 2022-04-11 at 7 24 30 PM" src="https://user-images.githubusercontent.com/120486/162796448-99ba22d4-0fed-4489-a258-0597414750a9.png"> | <img width="1440" alt="Screenshot 2022-04-11 at 7 29 09 PM" src="https://user-images.githubusercontent.com/120486/162796450-504ffa30-78fe-4b07-9e48-586cd6962de5.png"> |